core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation-dropdown.directive.html

```diff
@@ -0,0 +1,151 @@
+<div class="dropdown hidden-md hidden-lg ng-cloak" style="position: absolute; right: 100px; top:10px;">
```

Comment:
I'm surprised to see so much added code in this PR with little in the way of corresponding removals. What happened to the old HTML/JS code -- is it still needed/used?

Note that code should not be duplicated. If this code mimics existing code significantly then we should find a way to deduplicate it, otherwise it becomes very hard to maintain.

Reply:
I had to make few changes to the original code like adding the wordings(option names),appropriate styling and a few other changes inside the code. So I am not sure if you can deduplicate the code.
The old HTML/JS code is still used when the screen is in the normal size and it is needed.